code/modules/mining/mineral_effect.dm

```diff
@@ -15,6 +15,7 @@
 	ore_key = M.name
 	SetIconState("rock_[ore_key]")
 	var/turf/T = get_turf(src)
+	T.overlays += image('icons/obj/mining.dmi', "rock_[ore_key]", dir = 1)
```

Comment:
I'm pretty sure ores not showing up was intended, but mining needs all the help it can get.